modules/distribution-zones/src/main/java/org/apache/ignite/internal/distributionzones/DistributionZoneManager.java

```diff
@@ -301,8 +303,6 @@ public CompletableFuture<Void> startAsync(ComponentContext componentContext) {
             // fires CatalogManager's ZONE_CREATE event, and the state of DistributionZoneManager becomes consistent.
             int catalogVersion = catalogManager.latestCatalogVersion();
 
-            partitionDistributionResetTimeoutConfiguration.init();
```

Comment:
It's not directly related to your PR. 

1. What does `SystemDistributedConfigurationPropertyHolder#init` does?
2. What will happen if it's not called?

Reply:
I'd rather expect configuration engine to "call init" on all system properties rather than doing that by property owner. What if there are multiple owners? Currently it's easier to forget to call init, etc.

Reply:
Current holder is a tiny wrapeer for one caller around one configuration property. Nothing more. We already have a discussion with Ivan about the extension to more generic mechanism with the listeners aggregation and etc. - I think we should do it there

Reply:
Yep, it's definitely not related to the current PR.